lib/gcpspanner/feature_search_query.go

```diff
@@ -104,11 +104,24 @@ func (b *FeatureSearchFilterBuilder) traverseAndGenerateFilters(node *searchtype
 			filterString := strings.Join(childFilters, joiner)
 
 			if strings.TrimSpace(filterString) != "" {
-				filters = append(filters, "("+filterString+")")
+				filters = append(filters, filterString)
 			}
 
 		}
 
+	case node.Keyword == searchtypes.KeywordParens:
+		// Handle parenthesized sub-expressions.
+		var childFilters []string
+		for _, child := range node.Children {
+			childFilters = append(childFilters, b.traverseAndGenerateFilters(child)...)
+		}
+
+		filter := strings.Join(childFilters, " ")
```

Comment:
Would you mind explaining why space is needed here?

Reply:
Good question: I initially added to be safe but I see that if there are multiple terms in the parenthesis, they will fall under a keyword node. And given the keyword node already [adds spaces](https://github.com/GoogleChrome/webstatus.dev/blob/10c5f9dea62e0d3a19da7f2a17346d2b00617e74/lib/gcpspanner/feature_search_query.go#L100-L103), we don't need the space. Good catch!

I also will add another test case to help illustrate that too.